shared/src/store/action-reducers/material.ts

```diff
@@ -23,8 +25,23 @@ export namespace MaterialActionReducers {
         action: MoveMaterialAction,
         reducer: (draftState, { materialId, targetPosition }) => {
             const material = getElement(draftState, 'materials', materialId);
+
+            const startPosition = material.position;
+
+            if (startPosition === undefined) {
+                throw new ReducerError(
+                    `Material with id ${material.id} can't be moved, as its position is undefined`
+                );
+            }
```

Comment:
This is a breaking change as previously one could just call this reducer to move material from a vehicle to the map.